### PR TITLE
Fix typos and introduce a workflow

### PR DESCRIPTION
**Pull request by szepeviktor**
Found a few misspellings and prevent future ones.

### CODE REVIEW

**Comment by Rua** [comment]
There's no point in fixing vk.xml, because that's generated by the Vulkan project itself. It would be overwritten as soon as Vulkano updates to a new Vulkan version.

**Comment by szepeviktor** [comment]
Thank you. I didn't know that.

File: .typos.toml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+[files]
+extend-exclude = [
+    ".git/",
+    "vulkano/spirv.core.grammar.json",
+    "vulkano/vk.xml",
+]
+ignore-hidden = false
+
+[default]
```
Comment (Rua) [inline .typos.toml:9]: Is this needed?

Comment (szepeviktor) [reply .typos.toml:9]: Yes. That is the section for `extend-ignore-re`.

Comment (Rua) [reply .typos.toml:9]: Is it not possible to tell it to ignore the whole file, rather than specific names within the file?

Comment (szepeviktor) [reply .typos.toml]: Oh! I am sorry.
Those two name are in vk.xml.
Fixed.

**Comment by szepeviktor** [comment]
Glad to contribute.

**Pull request merged**